eng/pipelines/libraries/base-job.yml

```diff
@@ -75,6 +75,10 @@ jobs:
           - _finalFrameworkArg: -allConfigurations
           - _extraHelixArguments: /p:BuildAllConfigurations=true
 
+        - ${{ if eq(parameters.framework, 'sourceBuild') }}:
```

Comment:
It seems hacky to use `sourceBuild` as a framework. Just use `isSourceBuild` parameter as in other places.